### PR TITLE
feat: [SC-25607] Remove Tooltip component from @namehash/nameguard-react

### PR DESCRIPTION
**Pull request by FrancoAguzzi**

### CODE REVIEW

File: apps/nameguard.io/src/components/molecules/HeroStartCommand.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 "use client";
 
 import { useEffect, useState } from "react";
-import { Tooltip } from "@namehash/nameguard-react";
+import { Tooltip } from "@namehash/namekit-react";
```
Comment (lightwalker-eth) [inline apps/nameguard.io/src/components/molecules/HeroStartCommand.tsx:4]: This is broken. Please ensure you test all changes yourself before asking for a PR review.

CleanShot 2024-08-25 at 17 35 42

**Review by lightwalker-eth — CHANGES_REQUESTED** [review]
@FrancoAguzzi This change broke all the tooltips in each file changed in this PR. Please be sure you carefully review your own work before sending it for PR review.

File: apps/nameguard.io/src/components/molecules/HeroStartCommand.tsx (hunk 1 above)
Comment (FrancoAguzzi) [reply apps/nameguard.io/src/components/molecules/HeroStartCommand.tsx:4]: @lightwalker-eth fixed

File: packages/nameguard-react/src/components/UnknownReportIcon/UnknownReportIcon.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ export const UnknownReportIcon = ({
 }: UnknownShieldProps) => {
   return (
     <Tooltip
-      trigger={RatingUnknownIcon({
+      trigge={RatingUnknownIcon({
```
Comment (lightwalker-eth) [inline packages/nameguard-react/src/components/UnknownReportIcon/UnknownReportIcon.tsx:25]: @FrancoAguzzi  Not sure about this change? Looks like a typo?

These all seem broken.

Have you tested your work here?

CleanShot 2024-08-27 at 20 20 13

**Review by lightwalker-eth — CHANGES_REQUESTED** [review]
@FrancoAguzzi Just one issue found.

File: packages/nameguard-react/src/components/UnknownReportIcon/UnknownReportIcon.tsx (hunk 2 above)
Comment (notrab) [inline packages/nameguard-react/src/components/UnknownReportIcon/UnknownReportIcon.tsx:25]:
```suggestion
      trigger={RatingUnknownIcon({
```

Comment (FrancoAguzzi) [reply packages/nameguard-react/src/components/UnknownReportIcon/UnknownReportIcon.tsx:25]: It is a typo!

**Review by lightwalker-eth — APPROVED** [review]
Looks good :+1:

**Pull request merged**